Alao-Abiodun/Zigara-backend
# feature/endpoint-to-scheule-a-service

### What does the PR do?
- Endpoint for scheduling services.

### How to test manually
- git clone repo (Kindly ignore if that done before) then git pull upstream develop to update your own code base.
- enter cmd ```npm install``` or ```yarn install``` and then ```npm run dev``` or ```yarn dev``` depending on the package-name.
- Launch your postman.

GOTO  ``` {{baseUrl}}/service/schedule ``` where baseUrl is the ```http:localhost:port-number/service/schedule```.

You will test for when the service type is ```pickup_and_delivery``` and when the service type is ```packing_and_moving```

![packing_and_moving](https://user-images.githubusercontent.com/46402042/163157669-befb8f9f-0741-4569-8685-488c59943c15.jpg)
![pickup_and_delivery](https://user-images.githubusercontent.com/46402042/163157675-a9817584-c6e3-419b-81bb-d019a75c2d0e.jpg)

And you will get the expected response when the endpoint is send.
